src/App.vue

```diff
@@ -29,20 +29,20 @@ export default defineComponent({
         header: this.$t("Change time zone"),
         message: this.$t('Would you like to update your time zone to . Your profile is currently set to . This setting can always be changed from the settings menu.', { localTimeZone: payload.localTimeZone, profileTimeZone: payload.profileTimeZone }),
         buttons: [
-            {
-              text: this.$t("Dismiss"),
-              role: 'cancel',
-              cssClass: 'secondary'
+          {
```

Comment:
Please revert all the unrelated indentation changes

Reply:
Reverted indentation changes.